apps/api/src/evm/protocols/governorBravo/writers.ts

```diff
@@ -230,8 +230,6 @@ export function createWriters(
     rawEvent,
     helpers
   }) => {
-    console.log('Handle proposal created');
```

Comment:
Don't we want to know about this, or is this replaced?

Reply:
Wasn't sure if we should keep those or not, added them back with logger.

Reply:
Yeah, we kind of already does that (with `found contract event`) but we have some conditions that might prevent from writer from actually doing it's work (no txId, no event, no rawEvent) so maybe it might be helpful.

Reply:
If no even or rawEvent we wont see the log now that you moved it after the check